parm/post_avblflds.xml

```diff
@@ -4282,9 +4282,9 @@
 
     <param>
        <post_avblfldidx>487</post_avblfldidx>
-       <shortname>GSD_PRES_ON_CLOUD_BASE</shortname>
-       <pname>PRES</pname>
-       <fixed_sfc1_type>cloud_base</fixed_sfc1_type>
+       <shortname>GSD_EXP_CEILING</shortname>
+       <pname>CEIL</pname>
+       <fixed_sfc1_type>ceiling</fixed_sfc1_type>
```

Comment:
@EricJames-NOAA The "ceiling" is not a standard grib2 surface type. Would you consider "cloud_ceilng"? You may see the reference at 
https://github.com/NOAA-EMC/NCEPLIBS-g2tmpl/blob/develop/src/grib2_all_tables_module.f90#L307